locales/en/moves.json

```diff
@@ -80,5 +80,14 @@
       "confirmation": "Are you sure you want to cancel the move for <strong>{{name}}</strong> to <strong>{{location}}</strong>?",
       "warning": "This will delete this move request."
     }
+  },
+  "confirmation": {
+    "page_title": "Move confirmation for {{name}}",
+    "panel": {
+      "heading": "Move scheduled",
+      "content": "Move reference<br><strong>{{moveReference}}</strong>"
+    },
+    "detail": "The move for <strong>{{fullname}}</strong> to <strong>{{location}}</strong> on <strong>{{date}}</strong> has been scheduled",
```

Comment:
`name` instead of `fullname` to be consistent with above and elsewhere? The translation doesn't care what part of a name it is, that's more the concern of what is sending it to the translation.